frontend/src/app/modules/interventions/details/details.component.html

```diff
@@ -35,4 +35,31 @@
       </agm-map>
     </section>
   </mat-card-content>
+  <div>
+    <div *ngFor="let comment of intervention.comments">
+      <div class="comment">
+        <div class="comment-box">
+          <div class="comment-text">{{ comment.comment }}</div>
+          <div class="comment-footer">
+            <div class="comment-info">
+              <span class="comment-author">
+                <em>Anonymous</em>
+              </span>
+              <span class="comment-date">{{ comment.createdDate | date: 'medium' }}</span>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="comment-form">
```

Comment:
I would also keep the markup here to a bare minimum if possible, e.g.
```
<form class="comment-form" [formGroup]="form" (ngSubmit)="addComment()" novalidate>
        <div class="form-row">
          <textarea class="input" formControlName="comment" placeholder="Add comment..." required></textarea>
        </div>
        <input type="submit" value="Add Comment" [disabled]="!form.valid" />
</form>
```

Reply:
I've simplified the HTML structure